docs/excel/excel-add-ins-error-handling.md

```diff
@@ -73,6 +74,7 @@ The following table is a list of errors that the API may return.
 |`MergedRangeConflict`|Cannot complete the operation. A table can't overlap with another table, a PivotTable report, query results, merged cells, or an XML Map.|
 |`NonBlankCellOffSheet`|Microsoft Excel can't insert new cells because it would push non-empty cells off the end of the worksheet. These non-empty cells might appear empty but have blank values, some formatting, or a formula. Delete enough rows or columns to make room for what you want to insert and then try again.|
 |`NotImplemented`|The requested feature isn't implemented.|
+|`OperationCellsExceedLimit`|The attempted operation affects more than the limit of 33554000 cells. <br><br>If the `TableColumnCollection.add API` triggers this error, confirm that there is no unintentional data within the worksheet but outside of the table. In particular, check for data in the right-most columns of the worksheet. Remove the unintended data to resolve this error. One way to verify how many cells that an operation processes is to run the following calculation: `(number of table rows) x (16383 - (number of table columns))`. The number 16383 is the maximum number of columns that Excel supports. <br><br>This error only occurs in Excel on the web.|
```

Comment:
Is this all part of the error message that is displayed to users? It's not bad (in fact, it's very descriptive), just making sure we're not hiding the solution in an unexpected place.

Reply:
Addressed in first comment on this PR, let me know what you think. 

![image](https://user-images.githubusercontent.com/7977793/134051377-17fd99cf-d5cf-4570-9e73-5fd3d8d6a3af.png)

Reply:
Ah, sorry to miss that. I think that's fine, though I wonder the expanded guidance my still get overlooked since the column is labelled "Error message". Does adding a third column called "Notes" mess up the page formatting too much?

Reply:
I just tested. I think neither strategy (new Notes column or leave this note in the Message column) is particularly better than the other. The Notes column has the benefit of being forward-looking — more error codes may have solution suggestions in the future and this gives us a place to put those notes. 

https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-error-handling?branch=pr-en-us-2933